### ISSUE TITLE
[Request] MSU-1 Support

### ISSUE THREAD

**Issue by pfruh** [bubble2k16/snes9x_3ds#23]
Would it be possible to add the MSU-1 support from snes9x to this port?

https://github.com/snes9xgit/snes9x/tree/msu1

**Comment by bubble2k16** [comment]
No plans for now.

**Comment by mer2329** [comment]
i would also like to request this, as this feature would get me to move from VC injecting, on over to a SNES emulator.

**Comment by littlelum** [comment]
Yeah, I'd like to request this too. It would be fun to play the BS Zelda games with full voice acting on the 3ds

**Comment by ScapeGoat43** [comment]
any luck?
